### PR TITLE
Improvements to api.md for 1.5 changes

### PR DESCRIPTION
**Pull request by cevian**

### CODE REVIEW

File: api.md [hunk 1]
```diff
@@ -937,138 +1021,101 @@ ALTER TABLE <table_name> SET (timescale.compress, timescale.compress_orderby
...
-the chunk first using this process.
+## Compress Chunk :community_function: [](compress_chunk)
 
->:TIP: Prior to decompressing chunks for the purpose of data backfill or updating you should
-first stop any compression policy that is active on the hypertable you plan to perform this
-operation on.  Once the update and/or backfill is complete simply turn the policy back on
-and the system will recompress your chucks.
+The compress_chunk function is used to compress a specific chunk. This is
+most often used instead of the
+[add_compress_chunk_policy](add_compress_chunk_policy) function, when a user
```
Comment (solugebefola) [inline api.md]:
```suggestion
[add_compress_chunk_policy](#add_compress_chunk_policy) function, when a user
```

File: api.md [hunk 2]
```diff
@@ -748,6 +748,42 @@ For a table with more than one space dimension:
 SELECT set_number_partitions('conditions', 2, 'device_id');
 ```
 
+
+--
+
+## set_integer_now_func() [](set_integer_now_func)
```
Comment (solugebefola) [inline api.md]: This function needs to be added to the command list at the top of the file

File: api.md [hunk 3]
```diff
@@ -884,51 +920,99 @@ SELECT reorder_chunk('_timescaledb_internal._hyper_1_10_chunk', 'conditions_devi
 
 runs a reorder on the `_timescaledb_internal._hyper_1_10_chunk` chunk using the `conditions_device_id_time_idx` index.
 
+--
+
+## move_chunk() :enterprise_function: [](move_chunk)
+TimescaleDB allows users to move data (and indexes) to alternative
+tablespaces. This allows the user the ability to move data to more cost
+effective storage as it ages. This function acts like the combination of the
+[PostgreSQL CLUSTER command](postgres-cluster) and the
```
Comment (solugebefola) [inline api.md]:
```suggestion
[PostgreSQL CLUSTER command][postgres-cluster] and the 
```

File: api.md [hunk 4]
```diff
@@ -884,51 +920,99 @@ SELECT reorder_chunk('_timescaledb_internal._hyper_1_10_chunk', 'conditions_devi
...
 runs a reorder on the `_timescaledb_internal._hyper_1_10_chunk` chunk using the `conditions_device_id_time_idx` index.
 
+--
+
+## move_chunk() :enterprise_function: [](move_chunk)
+TimescaleDB allows users to move data (and indexes) to alternative
+tablespaces. This allows the user the ability to move data to more cost
+effective storage as it ages. This function acts like the combination of the
+[PostgreSQL CLUSTER command](postgres-cluster) and the 
+[PostgreSQL ALTER TABLE...SET TABLESPACE command](postgres-altertable). However, it uses lower
```
Comment (solugebefola) [inline api.md]:
```suggestion
[PostgreSQL ALTER TABLE...SET TABLESPACE command][postgres-altertable]. However, it uses lower
```

File: api.md [hunk 5]
```diff
@@ -884,51 +920,99 @@ SELECT reorder_chunk('_timescaledb_internal._hyper_1_10_chunk', 'conditions_devi
...
 |Name|Description|
 |---|---|
-| `timescale.compress_orderby` | Use of order by to organize data within hypertable |
+| `timescaledb.compress` | Boolean to enable compression |
 
 #### Other Options [](compression_alter-table-other-options)
 |Name|Description|
 |---|---|
-| `timescaledb.compress_segmentby` | Use of segment by to organize data within hypertable |
+| `timescaledb.compress_orderby` | Order used by compression, specified in the same way that the ORDER BY clause in a SELECT query. The default is the descending order of the hypertable's time column. |
```
Comment (solugebefola) [inline api.md]:
```suggestion
| `timescaledb.compress_orderby` | Order used by compression, specified in the same way as the ORDER BY clause in a SELECT query. The default is the descending order of the hypertable's time column. |
```

File: api.md [hunk 6]
```diff
@@ -884,51 +920,99 @@ SELECT reorder_chunk('_timescaledb_internal._hyper_1_10_chunk', 'conditions_devi
...
+| `timescaledb.compress_segmentby` | Column list on which to key the compressed segments. An identifier representing the source of the data such as `device_id` or `tags_id` is usually a good candidate. The default is no segment by columns. |
```
Comment (solugebefola) [inline api.md]:
```suggestion
| `timescaledb.compress_segmentby` | Column list on which to key the compressed segments. An identifier representing the source of the data such as `device_id` or `tags_id` is usually a good candidate. The default is no `segment by` columns. |
```

Not sure about this suggestion, take it or leave it

File: api.md [hunk 7]
```diff
@@ -937,138 +1021,101 @@ ALTER TABLE <table_name> SET (timescale.compress, timescale.compress_orderby
...
-In this example we will compress chunk 1_2
-``` sql
-SELECT compress_chunk( '_timescaledb_internal._hyper_1_2_chunk');
-```
+## Add Compress Chunk Policy :community_function: [](add_compress_chunk_policy)
+Allows you to set a policy by which the system will compress a chunk
+automatically in the background after it reaches a given age.
 
-#### Required Arguments [](compress_chunk-required-arguments)
+#### Required Arguments [](add_compress_chunk_policy-required-arguments)
```
Comment (solugebefola) [inline api.md]: Add syntax section before this section

File: api.md [hunk 8]
```diff
@@ -937,138 +1021,101 @@ ALTER TABLE <table_name> SET (timescale.compress, timescale.compress_orderby
...
 
 |Name|Description|
 |---|---|
-| `chunk_name` | Name of the chunck to be compressed|
-
-## Compress Policy (Compression) :community_function: [](compress_policy)
-Allows you to set a policy  by which the system will compress a chunk
-automatically in the background after it reaches a given age.
+| `table_name` | (REGCLASS) Name of the the policy will act on.|
```
Comment (solugebefola) [inline api.md]:
```suggestion
| `table_name` | (REGCLASS) Name of the table that the policy will act on.|
```

File: api.md [hunk 9]
```diff
@@ -937,138 +1021,101 @@ ALTER TABLE <table_name> SET (timescale.compress, timescale.compress_orderby
...
+- For hypertables with TIMESTAMP, TIMESTAMPTZ, and DATE time columns: the time interval should be an INTERVAL type
+- For hypertables with integer-based timestamps: the time interval should be an integer type (this requires
+the [integer_now_func](set_integer_now_func) to be set).
 
-``` sql
-SELECT add_compress_chunks_policy('<table_name>', '<time interval>'::interval);
-```
-#### Sample Usage [](compress_policy-sample-usage)
+#### Sample Usage [](add_compress_chunk_policy-sample-usage)
 In this example we will compress chunks older than 60 days from the 'cpu' table.
```
Comment (solugebefola) [inline api.md]: Might as well reword this to match your other changes

File: api.md [hunk 10]
```diff
@@ -937,138 +1021,101 @@ ALTER TABLE <table_name> SET (timescale.compress, timescale.compress_orderby
...
 ``` sql
-SELECT remove_compress_chunks_policy('<table_name>');
+SELECT add_compress_chunks_policy('table_with_bigint_time', BIGINT '600000');
 ```
-#### Required Arguments [](compress_policy_remove-required-arguments)
+
+## Remove Compress Chunk Policy :community_function: [](remove_compress_chunk_policy)
+If you need to remove the compression policy. To re-start policy basd compression againn you will need to re-add the policy.
+
+#### Required Arguments [](remove_compress_chunk_policy-required-arguments)
```
Comment (solugebefola) [inline api.md]: Add syntax section before this one

File: api.md (hunk 7 above)
Comment (cevian) [reply api.md]: Leaving this as-is, as we discussed.

File: api.md (hunk 9 above)
Comment (cevian) [reply api.md]: done.

File: api.md (hunk 10 above)
Comment (cevian) [reply api.md]: Leaving this as-is, as we discussed.

File: api.md (hunk 2 above)
Comment (cevian) [reply api.md]: done
